modules/qt/nixos.nix

```
@@ -28,6 +28,9 @@ in
   };
 
   config = lib.mkIf (config.stylix.enable && config.stylix.targets.qt.enable) {
+    warnings = lib.mkIf config.services.xserver.desktopManager.plasma6.enable [
+      "Styling of Plasma6 is not yet supported upstream. See https://github.com/nix-community/home-manager/issues/5098"
+    ];
```

Align with commit 5e8be7521e8f7cc927c92f24f8753e64ec050cdf:

```suggestion
    warnings =
      lib.optional
      config.services.xserver.desktopManager.plasma6.enable
      "stylix: qt: Plasma6 support is currently broken: https://github.com/nix-community/home-manager/issues/5098";
```